budget/web.py

```diff
@@ -66,6 +80,23 @@ def pull_project(endpoint, values):
                     url_for(".authenticate", project_id=project_id))
 
 
+@main.route("/authenticate_admin", methods=["GET", "POST"])
```

Comment:
Should the route be just `admin`? Otherwise, the URL could be joined with `-` rather than `_` (it seems more URLish to me)